packages/react-magma-dom/src/components/TreeView/useTreeItem.ts

```diff
@@ -420,28 +414,50 @@ export function useTreeItem(props: UseTreeItemProps, forwardedRef) {
       !topLevel &&
       !isDisabled
     ) {
-      setStatusUpdatedBy(StatusUpdatedByOptions.parent);
-      setCheckedStatus(parentCheckedStatus);
-      if (hasOwnTreeItems) {
-        if (getAllChildrenEnabled(treeItemChildren)) {
-          setChildrenCheckedStatus(
-            Array(numberOfTreeItemChildren).fill(parentCheckedStatus)
-          );
-        } else {
+      // TODO: this needs to be updated
+      const isChildOfSelectedItem = false;
```

Comment:
I found problem with build here.We don't use isChildOfSelectedItem. We need to remove (not commented) this variable.  I verified locally - build pass.